python/gtsam/gtsam.tpl

```diff
@@ -0,0 +1,30 @@
+{include_boost}
```

Comment:
What is this and where is this described? Maybe add something to readme about “inner workings”?

Reply:
Sure, will add.